### PR TITLE
add visualize_results_v2 in launch.py

### PR DESCRIPTION
**Pull request by yuchen-xue**
add visualize_results_v2 in launch.py

### CODE REVIEW

File: launch.py [hunk 1]
```diff
@@ -60,6 +60,8 @@ def main():
         tool_file = 'compute_tracks.py'
     elif args.mode == 'eval':
         tool_file = 'eval_mpii.py'
+    elif args.mode == 'vis':
+    tool_file = 'visualize_results_v2.py'
```
Comment (rohitgirdhar) [inline launch.py]: An indentation seems missing here..

**Review by rohitgirdhar — CHANGES_REQUESTED** [review]
Thanks! Please fix the indent

**Comment by yuchen-xue** [comment]
@rohitgirdhar Ok, I've fixxed it.

**Comment by SeanSyue** [comment]
@rohitgirdhar Thx, I've fixxed it.

**Review by rohitgirdhar — APPROVED** [review]

**Pull request merged**